WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -28,7 +29,7 @@ public partial class CoinJoinStateViewModel : ViewModelBase
 	private const string WaitingForBlameRoundMessage = "Awaiting the blame round";
 	private const string WaitingRoundMessage = "Awaiting a round";
 	private const string PlebStopMessage = "Coinjoin may be uneconomical";
-	private const string PlebStopMessageBelow = "Add more funds or press 'Play' to bypass";
+	public const string PlebStopMessageBelow = "Add more funds or press 'Play' to bypass";
```

Comment:
ultra nit: to have the same with line 23
maybe change if you retouch, or feel free to ignore
```suggestion
	public const string PlebStopMessageBelow = "Add more funds or press Play to bypass";
```